FirstProject/css/feedback.css

```diff
@@ -0,0 +1,162 @@
+/*Feedback HTML*/
+
+.footer_section {
+    padding: 25px 0 12px 0;
+    font-weight:bold;
+    font-size:16px;
+    color: gray;
+    text-align: right;
+}
+
+.feedback_interview_section {
+    text-align: center;
+    padding-top: 125px;
+    position: absolute;
+    width:100%;
+}
+
+.header_form_section {
+    width: 21%;
+    background-color: white;
+    height: 340px;
+}
+
+.header_background_section {
+    height: 640px;
+}
+
+h1.interview_text_area {
+    background-color: #03A9F4;
+    color: white;
+    font-size: 22px;
+    padding: 27px;
+}
+
+.username_area_input {
+    width: 80%;
+    margin-top: 30px;
+    border-right: none;
+    border-left: none;
+    border-top: none;
+}
+
+.password_area_input {
```

Comment:
This is identical to the styling for `.username_area_input`. Let's go ahead and combine the two selectors.